### PR TITLE
Add modulus operator support

### PR DESCRIPTION
**Pull request by MingchenZhang**
Add configurator side of https://github.com/iNavFlight/inav/pull/6699

### CODE REVIEW

File: js/fc.js [hunk 1]
```diff
@@ -1113,106 +1113,111 @@ var FC = {
                 output: "raw"
             },
             18: {
+                name: "MOD",
```
Comment (DzikuVx) [inline js/fc.js:1116]: Please add it a LAST entry not to break compatibility with existing solutions

**Review by DzikuVx — CHANGES_REQUESTED** [review]

File: js/fc.js (hunk 1 above)
Comment (MingchenZhang) [reply js/fc.js:1116]: I just moved the mod to the last entry.

**Review by DzikuVx — APPROVED** [review]

**Comment by avsaase** [comment]
Is it not possible to keep the ordering as originally intended by @MingchenZhang? It is kind of annoying if new operators can only be added at the end of the dropdown menu to avoid breaking compatibility with old diffs.

**Comment by DzikuVx** [comment]
@avsaase one point in a future we should do updated UI for the operands with grouping and ordering. But not now

**Pull request merged**